types/vuelidate/lib/validators.d.ts

```diff
@@ -42,10 +42,13 @@ export function between(min: number, max: number): ValidationRule
 export function alpha(): ValidationRule
 export function alphaNum(): ValidationRule
 export function numeric(): ValidationRule
+export function integer(): ValidationRule
+export function decimal(): ValidationRule
 export function email(): ValidationRule
 export function ipAddress(): ValidationRule
 export function macAddress(): ValidationRule
 export function sameAs(field: string | ((vm: any, parentVm?: Vue) => any)): ValidationRule
 export function url(): ValidationRule
-export function or(...validators: ValidationRule[]): ValidationRule
-export function and(...validators: ValidationRule[]): ValidationRule
+export function not(validator: ValidationRule | CustomRule): ValidationRule
```

Comment:
would it make sense to have something like "CustomRule extends ValidationRule"?

Reply:
hmm it might, let me dig into the package src and see how if that works (but if memory serves, it should).